Composer/packages/client/src/pages/botProject/PublishProfieWrapperDialog.tsx

```diff
@@ -18,25 +16,23 @@ import { PublishProfileDialog } from './create-publish-profile/PublishProfileDia
 type PublishProfileWrapperDialogProps = {
   projectId: string;
   onClose: () => void;
-  onOpen: () => void;
   onUpdateIsCreateProfileFromSkill: (isCreateProfileFromSkill: boolean) => void;
 };
 
 export const PublishProfileWrapperDialog: React.FC<PublishProfileWrapperDialogProps> = (props) => {
-  const { projectId, onClose, onOpen, onUpdateIsCreateProfileFromSkill } = props;
+  const { projectId, onClose, onUpdateIsCreateProfileFromSkill } = props;
   const { publishTargets } = useRecoilValue(settingsState(projectId));
   const { getPublishTargetTypes, setPublishTargets } = useRecoilValue(dispatcherState);
   const publishTypes = useRecoilValue(publishTypesState(projectId));
 
   const [showPublishProfileDialog, setShowPublishProfileDialog] = useState(false);
-  const [showAuthDialog, setShowAuthDialog] = useState(false);
 
   const [currentPublishProfile, setCurrentPublishProfile] = useState<{ index: number; item: PublishTarget } | null>(
     null
   );
 
   useEffect(() => {
-    isShowAuthDialog(true) ? setShowAuthDialog(true) : setShowPublishProfileDialog(true);
+    setShowPublishProfileDialog(true);
```

Comment:
every time this component it re-rendered you are showing the publish profile dialog, is that what you want?

Reply:
yes ... in fact, that whole check can probably be removed. It originally functioned to hide this modal until the token auth UI completed -- however, the token auth ui now happens later in the flow *inside the publishprofiledialog*